app/ui.js

```diff
@@ -944,17 +970,20 @@ const UI = {
     clipboardReceive(e) {
         Log.Debug(">> UI.clipboardReceive: " + e.detail.text.substr(0, 40) + "...");
         document.getElementById('noVNC_clipboard_text').value = e.detail.text;
+        UI.writeLocalClipboard(e.detail.text);
         Log.Debug("<< UI.clipboardReceive");
     },
 
     clipboardClear() {
         document.getElementById('noVNC_clipboard_text').value = "";
+        UI.writeLocalClipboard("");
         UI.rfb.clipboardPasteFrom("");
     },
 
     clipboardSend() {
         const text = document.getElementById('noVNC_clipboard_text').value;
         Log.Debug(">> UI.clipboardSend: " + text.substr(0, 40) + "...");
+        UI.writeLocalClipboard(text);
```

Comment:
Why writeLocalClipboard here?

Reply:
Thanks for pointing this out. I've recognized after that, that my code is still buggy in the case, if you change the clipboard in the "clipboard overlay textfield". I'll fix this in the next days.

Reply:
What was the bug you saw?

Seeing the new comment you added to the handler for 'input', my question regarding why you placed writeLocalClipboard was answered.

What's the reason for separating the sync to the 'input' event instead of the 'change' one?